README.md

```diff
@@ -636,6 +636,13 @@ This library partially supports [inline schema definition dereferencing]( http:/
 
 Note that it only supports local definition referencing, we do not plan on fetching foreign schemas over HTTP anytime soon. Basically, you can only reference a definition from the very schema object defining it.
 
+## JSON Schema supporting status
```

Comment:
You should run the `npm run build:readme` so this entry will be added to the TOC.